File: app/src/main/java/se/torgammelgard/pokertrax/MainApp.java
package se.torgammelgard.pokertrax;

import android.app.Application;

/**
 * TODO: Class header comment.
 */
public class MainApp extends Application {

    public DataSource mDataSource;

    @Override
    public void onCreate() {
        super.onCreate();

        mDataSource = new DataSource(getApplicationContext());

    }
}

File: app/src/main/java/se/torgammelgard/pokertrax/MainApp.kt
package se.torgammelgard.pokertrax

import android.app.Application
import se.torgammelgard.pokertrax.Database.DataSource

/**
 * Main application
 */
class MainApp : Application() {

    var mDataSource: DataSource?= null

    override fun onCreate() {
        super.onCreate()

        mDataSource = DataSource(applicationContext)

    }
}
